utils/keycodes.js

```diff
@@ -7,3 +7,4 @@ export const UP = 38;
 export const RIGHT = 39;
 export const DOWN = 40;
 export const DELETE = 46;
+export const LETTER_A = 65;
```

Comment:
Related: https://github.com/WordPress/gutenberg/pull/1215#issuecomment-309070463

Not sure what the issues are with IE11 and "non-standard key identifiers", also only available in latest iOS version, which violates [documented browser support](https://make.wordpress.org/design/handbook/design-guide/browser-support/).